packages/e2e-tests/src/tests/WrappedKeys.spec.ts

```diff
@@ -0,0 +1,1211 @@
+    ); // 10 mins expiry
+
+    const privateKey =
+      '4rXcTBAZVypFRGGER4TwSuGGxMvmRwvYA3jwuZfDY4YKX4VEbuUaPCWrZGSxujKknQCdN8UD9wMW8XYmT1BiLxmB'; // Already exists in the DB
+
+    try {
+      await importPrivateKey({
+        pkpSessionSigs: pkpSessionSigs!,
+        privateKey,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (
+        e.message.includes(
+          'Failed to make request for wrapped key: There is already a wrapped key stored with the same dataToEncryptHash. A wrapped key may only be associated with a single pkpAddress'
+        )
+      ) {
+        console.log('✅ THIS IS EXPECTED: ', e);
+      } else {
+        throw e;
+      }
+    }
+  });
+
+  it('Fail Import Wrapped Key Same PKP', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    console.log(pkpSessionSigs);
+
+    const privateKey1 = randomSolanaPrivateKey();
+
+    const { pkpAddress, id } = await importPrivateKey({
+      pkpSessionSigs: pkpSessionSigs!,
+      privateKey: privateKey1,
+      litNodeClient: devEnv.litNodeClient!,
+      publicKey: '0xdeadbeef',
+      keyType: 'K256',
+      memo: 'Test Key',
+    });
+
+    const alicePkpAddress = alice.authMethodOwnedPkp?.ethAddress;
+    expect(pkpAddress).toEqual(alicePkpAddress);
+    try {
+      const privateKey2 = randomSolanaPrivateKey();
+
+      await importPrivateKey({
+        pkpSessionSigs: pkpSessionSigs!,
+        privateKey: privateKey2,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (
+        e.message.includes(
+          'There is already a wrapped key stored, either for the provided pkpAddress, or with the same dataToEncryptHash; a pkpAddress may only have 1 wrapped key, and a wrapped key may only be associated with a single pkpAddress.'
+        )
+      ) {
+        console.log('✅ THIS IS EXPECTED: ', e);
+      } else {
+        throw e;
+      }
+    }
+  });
+
+  it('Fail Max Expiration Session Sig', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(devEnv, alice);
+
+    console.log(pkpSessionSigs);
+
+    try {
+      const privateKey = randomSolanaPrivateKey();
+
+      await importPrivateKey({
+        pkpSessionSigs: pkpSessionSigs!,
+        privateKey,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (e.message.includes('Expires too far in the future')) {
```

Comment:
We can use expect.toThrow() here instead, and :hocho: the extra `console.log`
See also https://jestjs.io/docs/expect#rejects :)